apps/vanilla/src/pages/2/src/views/roll-button-view.ts

```diff
@@ -0,0 +1,27 @@
+import { View } from './view';
+
+/** View of button that generate game turn. */
+export class RollButtonView extends View {
+	/** Html element. */
+	protected override element: HTMLElement;
+
+	public constructor() {
+		super();
+		this.element = this.getElement();
+	}
+
+	/** Returns template of view. */
+	public override getTemplate(): string {
+		return `<button class="primary-button game__roll-button">Make roll</button>`;
+	}
+
+	/**
+	 * Add events to button.
+	 * @param args Array of events and callbacks.
+	 */
+	public addEvents(...args: Array<[string, () => void]>): void {
```

Comment:
Please, try to stick with 1 approach for array type declaration
```suggestion
	public addEvents(...args: [string, () => void][]): void {
```